### ISSUE TITLE
Issues converting spigot BB code to Hangar code

### ISSUE THREAD

**Issue by PeachesMLG** [PaperMC/Hangar#291]
Tried to convert spigot bb code to hangar code
https://hangar.minidigger.me/Peaches_MLG/IridiumSkyblock
https://www.spigotmc.org/resources/iridium-skyblock-1-8-1-16.62480/
It messed up a bit

**Comment by Machine-Maker** [comment]
I believe the [YoutubeLink](https://github.com/vsch/flexmark-java/wiki/Extensions#youtube-embedded-link-transformer) extension is what we are looking for here.

**Comment by MiniDigger** [comment]
I guess this should be good to go now?

**Comment by kennytv** [comment]
right, fixed with https://github.com/HangarMC/Hangar/commit/6b3cd043840f136cb0f7896035d95eb3eacdc1b6
